vue.config.js

```diff
@@ -58,6 +58,13 @@ module.exports = {
         pathRewrite: {
           '/local-keycloak-config-url': ''
         }
+      },
+      // this is needed to avoid a PAYBC Not Found error when running locally
+      '/status/PAYBC': {
+        target: 'https://status-api-dev.apps.silver.devops.gov.bc.ca/api/v1/status/PAYBC',
```

Comment:
👍

Reply:
Did they say how for long these urls will be changed?

Reply:
They did not say, but as these are back-end URLs, they don't need to be vanity URLs and could just stay like that, I think.